src/org/traccar/model/Position.java

```diff
@@ -57,10 +58,11 @@
     public static final String KEY_THROTTLE = "throttle";
     public static final String KEY_MOTION = "motion";
     public static final String KEY_ARMED = "armed";
-    public static final String KEY_ACCURACY = "accuracy";
     public static final String KEY_GEOFENCE = "geofence";
     public static final String KEY_ACCELERATION = "acceleration";
     public static final String KEY_DEVICE_TEMP = "deviceTemp";
+    public static final String KEY_OPERATOR = "operator";
+    public static final String KEY_COMMAND = "command";
```

Comment:
In my review, KEY_TYPE seems to indicate "message type".  7 protocols send KEY_TYPE and 6 protocols send KEY_COMMAND.  TeltonikaProtocol sends both KEY_TYPE and KEY_COMMAND.  I am not familiar enough with the protocols that use these two keys.  Given that the 6 protocols used a hardcoded string "command" and not Position.KEY_TYPE, I recommend leaving it until you or someone else determines that they are indeed the same thing.